perl/p5-pdl/Portfile

```diff
@@ -4,9 +4,8 @@ PortSystem          1.0
 PortGroup           perl5 1.0
 
 perl5.branches      5.26
-perl5.setup         PDL 2.018
-revision            3
-maintainers         peculier.com:mgrimes openmaintainer
+perl5.setup         PDL 2.019
+maintainers         {peculier.com:mgrimes @mvgrimes}  openmaintainer
```

Comment:
Looks like there's an extra space before `openmaintainer` but maybe I'm being too picky. Overall, LGTM.

Reply:
Even _port lint --nitpick_ missed that one.  Thanks.